src/Screens/Admin/Category/Category.js

```diff
@@ -12,37 +12,38 @@ type Props = {
   category: { attributes?: Array<any>, title?: string },
   getCategory: string => Promise<void>
 };
-class Category extends Component<Props> {
-  UNSAFE_componentWillMount() {
-    const { id, getCategory } = this.props;
-    getCategory(id);
-  }
 
-  render() {
-    const { category } = this.props;
+const Category = ({
+  id,
+  category,
+  getCategory
+}: Props) => {
+  
+  useEffect(() => {
```

Comment:
Empty dependency lists and then using the dependencies trigger warnings. There are multiple warnings.

Please verify that it's safe to use and if not maybe use `useCallback()`?
https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies

Reply:
This part is done